publish.gradle

```diff
@@ -95,6 +95,12 @@ ext.configurePublish = { jnaForTestConfiguration = null ->
                 // Can't publish Javadoc yet: fxaclient isn't well behaved.
                 // artifact javadocJar
 
+                if (isMegazord) {
+                    artifact file("${projectDir}/../DEPENDENCIES.md"), {
+                        extension "LICENSES.md"
+                    }
```

Comment:
This just generates a separate file in the maven publication, like `lockbox-megazord-0.36.0.LICENSES.md`. I'm not sure it's that useful to consumers, but it's a start and we can iterate based on consumer feedback.